common/test/acceptance/pages/lms/courseware.py

```diff
@@ -128,6 +128,36 @@ def start_proctored_exam(self):
         # elf.wait_for_element_presence(".proctored-exam-code", "unique exam code")
 
     @property
+    def entrance_exam_status_msg(self):
+        """
+        Return the entrance exam status message selector on the top of courseware page.
+        """
+        return self.q(css='#content .container section.course-content .sequential-status-message')
+
+    @property
+    def is_not_passed_message_present(self):
+        """
+        Returns boolean indicating presence of not passed message.
+        """
+        return self.entrance_exam_status_msg.is_present() \
+            and "To access course materials, you must score" in self.entrance_exam_status_msg.text[0]
+
+    @property
```

Comment:
Ah, I see they are actually two different messages...hmm...I wonder if there is a clearer/simpler way to do this...